src/main/java/org/example/Ships/Main.java

```diff
@@ -0,0 +1,116 @@
+package org.example.Ships;
+
+import java.util.Scanner;
+
+public class Main {
+
+    public static void main(String[] args) {
+        menu();
+
+
+    }
+
+    public static void menu() {
```

Comment:
tu jest za duzo kodu w jednej funkcji/klasie :D jezeli menu trzyma rozmiar statku to nie jest to na pewno jedna odpowiedzialnosc